build.gradle.kts

```diff
@@ -58,23 +67,24 @@ sourceSets {
 
 dependencies {
     implementation(kotlin("stdlib-jdk8"))
-    implementation("org.apache.commons:commons-lang3:3.13.0")
+    implementation("org.apache.commons:commons-lang3:3.12.0")
     implementation("org.apache.commons:commons-text:1.10.0")
     implementation("org.apache.commons:commons-collections4:4.4")
+    implementation("commons-io:commons-io:2.15.0")
     implementation("com.opencsv:opencsv:5.8")
-    implementation("org.flowable:flowable-engine:6.7.2")
-    implementation("org.flowable:flowable-cdi:6.7.2")
-    implementation("org.flowable:flowable-cmmn-engine:6.7.2")
-    implementation("org.flowable:flowable-cmmn-cdi:6.7.2")
-    implementation("org.flowable:flowable-cmmn-engine-configurator:6.7.2")
+    implementation("org.flowable:flowable-engine:7.0.0")
+    implementation("org.flowable:flowable-cdi:7.0.0")
+    implementation("org.flowable:flowable-cmmn-engine:7.0.0")
+    implementation("org.flowable:flowable-cmmn-cdi:7.0.0")
+    implementation("org.flowable:flowable-cmmn-engine-configurator:7.0.0")
     implementation("org.slf4j:slf4j-jdk14:2.0.3")
     implementation("com.auth0:java-jwt:4.4.0")
     implementation("javax.cache:cache-api:1.1.1")
     implementation("com.google.guava:guava:32.1.3-jre")
-    implementation("com.mailjet:mailjet-client:5.2.5")
+    implementation("com.mailjet:mailjet-client:5.2.4")
     implementation("org.flywaydb:flyway-core:9.22.3")
     implementation("org.apache.solr:solr-solrj:9.4.0")
-    implementation("net.sf.webdav-servlet:webdav-servlet:2.0")
+    implementation("net.sf.webdav-servlet:webdav-servlet:3.0.0-INFONL")
```

Comment:
I would've expected another group name `nl.info` or the like

Reply:
Maybe in future yes for sure. But definitely not for this sprint. First we need to decide how to proceed with this webdav-servlet project. Maybe we just want to take the whole thing over from the original author. Don't know yet.

Reply:
I have added it to: https://dimpact.atlassian.net/wiki/spaces/PZW/pages/14123009/ZAC+WildFly+26+to+30+upgrade+issues+and+workarounds#Take-ownership-of-webdav-servlet-project-and-make-it-more-robust